endpoints/openrtb2/sample-requests/account-required/with-acct.json

```diff
@@ -0,0 +1,67 @@
+{
+    "description": "This request comes with no account id",
+    "message": "Invalid request: Prebid-server has been configured to discard requests that don't come with an Account ID. Please reach out to the prebid server host.\n",
+  
+    "requestPayload": {
+      "id": "some-request-id",
+      "site": {
+        "publisher": { "id": "not_bad_acct"},
+      	"page": "test.somepage.com"
+      },
+      "user": { },
+      "imp": [
+        {
+        	"id": "my-imp-id",
+        	"banner": {
```

Comment:
Can you please fix the formatting on this one? Thanks!

Reply:
Done